flow/queue/server_websocket.py

```diff
@@ -0,0 +1,216 @@
+import asyncio
+import websockets
+import util
+import db_functions
+import exceptions
+import datetime
+from typing import Dict, List, Set
+
+
+class Group:
+	def __init__(self, group_id: str, server: 'QueueWebsocketServer'):
+		self.group_id = group_id
+		self.server = server
+		self.listeners = set()  # type: Set[ConnectedClient]
+		self.queue = []  # type: List[QueueTicket]
+
+	async def add_listener(self, listener):
+		self.listeners.add(listener)
```

Comment:
Nope.

Listener may not have been the best name, it's adding someone who wants to be updated on who is in the queue when someone leaves/joins the queue.